documentation/general/general_setup.md

```diff
@@ -184,6 +184,19 @@ If not, you can use one of the many
 [installation methods](https://docs.docker.com/engine/install/) officially
 supported by Docker.
 
+Note that the building images for the frontend needs an advanced build feature,
+namely additional build contexts, that is provided by
+[Buildx](https://docs.docker.com/build/architecture/#buildx). Buildx is included
+in Docker Desktop and all
+[installation methods](https://docs.docker.com/engine/install/) for Docker
+Engine include steps to also install the Buildx plugin `docker-buildx-plugin`.
+
+```{tip}
+If it is not already, you should
+[make Buildx the default](https://github.com/docker/buildx#set-buildx-as-the-default-builder)
+builder.
+```
```

Comment:
Why is this?

Reply:
Because Buildx has more features compared to the legacy build system. Case in point: additional build contexts.

Reply:
But why should an Openverse contributor do this? I haven't done it and haven't run into any issues. Is it a requirement of the project? This tip has strong language "should" but isn't clear as to the _reason_, and considering it has system-wide implications (setting a different default), it should have a good reason to back it up. Other projects the person reading contributes to may not work with buildx, for example.

Reply:
Buildx is backwards compatible so using it as the default will not negatively affect any existing workflows.

Reply:
Anyway, Buildx is the default on engine v23+ and so I've removed that tip from the docs.